easybuild/easyconfigs/g/GROMACS/GROMACS-2019.3-foss-2019a.eb

```diff
@@ -0,0 +1,51 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2016 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+# Authors::
+# * Wiktor Jurkowski <wiktor.jurkowski@gmail.com>
+# * Fotis Georgatos <fotis@cern.ch>
+# * George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Adam Huffman <adam.huffman@crick.ac.uk>
+# License::   MIT/GPL
+# 2019.3 version
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+#
+
+name = 'GROMACS'
+version = '2019.3'
+
+homepage = 'http://www.gromacs.org'
+description = """
+GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
+"""
```

Comment:
I would suggest to keep the `This is a CPU only build, containing both MPI and threadMPI builds.` present in `GROMACS-2019-foss-2018b.eb`. This could be useful for less experienced users.